contracts/truefi2/CollateralVault.sol

```diff
@@ -60,6 +60,7 @@ contract CollateralVault is ICollateralVault, UpgradeableClaimable {
         );
         stakedAmount[msg.sender] = stakedAmount[msg.sender].add(amount);
         stakedToken.safeTransferFrom(msg.sender, address(this), amount);
+        lineOfCreditAgency.updateAllCreditScores(msg.sender);
```

Comment:
This can be pretty gas expensive. Should we guard it behind an if (borrowingMutex.locker() == LOCA). Same with unstake.

We can take this discussion to a follow-up PR since it's not required for audit.